paper.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Introduction to deep learning: Carpentries-style hands-on lesson material for introducing researchers to deep learning'
+tags:
+  - Python
+  - deep learning
+  - machine learning
+  - Keras
+  - neural networks
+authors:
+  - name: Sven A. van der Burg
+    orcid: 0000-0003-1250-6968
+    affiliation: 1 # (Multiple affiliations must be quoted, like "1, 2")
+  - name: Anne Fouilloux
+    orcid: 0000-0002-1784-2920
+    affiliation: 2
+  - name: Florian Huber
+    orcid: 0000-0002-3535-9406
+    affiliation: "1, 3"
+  - name: Dafne van Kuppevelt
+    affiliation: 1
+  - name: Peter Steinbach
+    orcid: 0000-0002-4974-230X
+    affiliation: 4
+  - name: Berend Weel
+    orcid: 0000-0002-9693-9332
+    affiliation: 1
+  - name: Colin Sauze
+    orcid: 0000-0001-5368-9217
+    affiliation: 5
+  - name: Samantha Wittke
+    orcid: 0000-0002-9625-7235
+    affiliation: "6,7"
+  - name: Djura Smits
+    orcid: 0000-0003-4096-0260
+    affiliation: 1
+  - name: Cunliang Geng
+    orcid: 0000-0002-1409-8358
+    affiliation: 1
+  - name: Pranav Chandramouli
+    orcid: 0000-0002-7896-2969
+    affiliation: 1
+
+affiliations:
+ - name: Netherlands eScience Center, Amsterdam, The Netherlands
+   index: 1
+ - name: Simula Research Laboratory, Oslo, Norway
+   index: 2
+ - name: Düsseldorf University of Applied Sciences, Düsseldorf, Germany
+   index: 3
+ - name: Helmholtz-Zentrum, Dresden-Rossendorf, Germany
+   index: 4
+ - name: National Oceanography Centre, Liverpool, Great-Britain
+   index: 5
+ - name: CSC - IT center for Science, Espoo, Finland
+   index: 6
+ - name: Aalto University, Espoo, Finland
+   index: 7
+date: 8 August 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+This article describes a hands-on introduction to the first steps in deep learning, 
+intended for researchers who are familiar with (non-deep) machine learning.
+
+The use of deep learning has seen a sharp increase of popularity and applicability over the last decade. 
+While deep learning can be a useful tool for researchers from a wide range of domains, 
+taking the first steps in the world of deep learning can be somewhat intimidating. 
+This introduction aims to cover the basics of deep learning in a practical and hands-on manner, 
+so that upon completion, students will be able to train their first neural network and understand 
+what next steps to take to improve the model.
+
+The lesson starts with explaining the basic concepts of neural networks, 
+and then go through the different steps of a deep learning workflow. 
+Learners will learn how to prepare data for deep learning, 
+how to implement a basic deep learning model in Python with Keras, 
+how to monitor and troubleshoot the training process and how to implement different layer types such as convolutional layers.
+
+# Statement of Need
+There are many great free online course materials on deep learning, 
+for example: fast.ai's Practical Deep Learning for Coders, MORE ONLINE COURSES
```

Comment:
No personal experience with free DL courses, but maybe some of these could be added: https://medium.com/javarevisited/10-free-deep-learning-courses-for-beginners-37b5de61f8dc ?

Reply:
Could also mention that many computing centers have DL courses available? [Example from CSC](https://ssl.eventilla.com/event/8aPek), which used to be free when it was funded by PRACE, which also had DL courses from other centers if I remember correctly.

Reply:
A classic is also the online courses with Andrew Ng (on Coursera) --> https://www.coursera.org/specializations/deep-learning

Reply:
Andrew Ng's courses are excellent (I have done those.) but they are not free.

Reply:
You are right, is it not free (at least not fully).
Coursera started to hide this a bit more I think, but those courses are free to "audit", so all videos and documents are freely accessible, but not the quizzes and code exercises.

Reply:
Great suggestions! I added them in f982c7cdb55d2de72229d9cdc4841aafb1334206 , will convert to references later.